Sources/TensorFlow/Loss.swift

```diff
@@ -24,6 +24,23 @@ public func meanSquaredError<Scalar: TensorFlowFloatingPoint>(
     return (expected - predicted).squared().mean()
 }
 
+/// Returns the mean squared logarithmic error between predictions and expectations.
+///
+/// - Note: Negative Tensor entries will be clamped at 0 to avoid undefined
+///   logarithmic behavior, as log() is undefined for negative reals.
```

Comment:
Swift functions are identified by their full name (base name + argument labels), so we should refer to them by their full name in documentation comments and wrap such references in backquotes. All of this is for having the best quality API documentation :)
```suggestion
///   logarithmic behavior, as `log(_:)` is undefined for negative reals.
```